samza-sql/src/main/java/org/apache/samza/sql/data/SamzaSqlRelMessage.java

```diff
@@ -64,6 +77,7 @@ public SamzaSqlRelMessage(List<String> fieldNames, List<Object> fieldValues) {
     }
     this.key = key;
     this.samzaSqlRelRecord = new SamzaSqlRelRecord(fieldNames, fieldValues);
+    this.samzaSqlRelMsgMetadata = new SamzaSqlRelMsgMetadata(Instant.now().toString());
```

Comment:
Would it better for the caller to decide on what time to put here ? IOW, should we make this constructor private ? Given that this is core datastructure in Samza SQL code, I would avoid having too many public constructors. We should also strive to make SamzaSqlRelMessage to not have any more smarts that it already has.